Editstina 904 (#4)
mana-ehlers/FearBase_manuscript: FearBase_manuscript.Rmd

**Introduction**

#Intro in data sharing

> Data sharing is increasingly being advocated by stakeholders, politics, scientific societies (Gollwitzer et al., 2020) and researchers alike and mandated by an increasing number of journals and funding agencies. The advantages of making data publicly available are obvious and have been outlined in several excellent sources (e.g. @Gilmore(2018), @Wilkinson(2016), @Klein et al., 2018): In brief advantages include the optimal use of scarce resources (manpower, money), improved reproducibility, acceleration of scientific progress through secondary use of data to answer new research questions, facilitated data aggregation across studies and labs and thus theory development and refinement. Data sharing can even benefit researchers careers through catalyzing new collaborations (@Popki(2019, career feature in nature)). Yet, these advantages of publicly sharing data can only be utilized when the shared data are indeed functional. A number of challenges sharing data and re-using publicly available data for re-analysis have been discussed previously in psychological science and cognitive neurocience (@Houtkoop(2018)). Challenges in sharing data include for instance the de-identification of (sensitive) data as well as the practical implementation of sharing data and code in which researchers have typically not received any training (@Houtkoop et al., 2018). Re-using publicly available data is made difficult by for instance proprietary formats of data files that limit accessibility, missing or insufficient (meta-) information rendering the data uninterpretable as well as idiosyncratic data structures, variable nomenclature and data formats (@Rahnev( et)2020), @Towse(2020)). As a consequence re-using publicly available data for secondary analyses is often highly time-consuming and not seldom involves a considerable amount of interaction between the original authors and the re-using researcher. 

Reviewer: ```suggestion
Data sharing is increasingly being advocated by stakeholders, politics, scientific societies (Gollwitzer et al., 2020) and researchers alike and mandated by an increasing number of journals and funding agencies. The benefits of making data publicly available are obvious and have been outlined in several excellent sources (e.g. @Gilmore(2018), @Wilkinson(2016), @Klein et al., 2018): In brief advantages include the optimal use of scarce resources (manpower, money), improved reproducibility, acceleration of scientific progress through secondary use of data to answer new research questions, facilitated data aggregation across studies and labs and thus theory development and refinement. Data sharing can even benefit researchers careers through catalyzing new collaborations (@Popki(2019, career feature in nature)). Yet, these advantages of publicly sharing data can only be utilized when the shared data are indeed functional. A number of challenges sharing data and re-using publicly available data for re-analysis have been discussed previously in psychological science and cognitive neurocience (@Houtkoop(2018)). Challenges in sharing data include for instance the de-identification of (sensitive) data as well as the practical implementation of sharing data and code in which researchers have typically not received any training (@Houtkoop et al., 2018). Re-using publicly available data is made difficult by for instance proprietary formats of data files that limit accessibility, missing or insufficient (meta-) information rendering the data uninterpretable as well as idiosyncratic data structures, variable nomenclature and data formats (@Rahnev( et)2020), @Towse(2020)). As a consequence re-using publicly available data for secondary analyses is often highly time-consuming and not seldom involves a considerable amount of interaction between the original authors and the re-using researcher. 
```

> Currently, even though data sharing is still relatively rare (REF NEEDED) the scientific community is currently not using the large, existing, open data to its full potential. We simply do not know what kind of data exists out there, what it contains, or how to use it without a huge investment with respect to time and manpower. 

Reviewer: ```suggestion
Currently, even though data sharing is still relatively rare (Houtkoop et al., 2018) the scientific community is currently not using the large, existing, open data to its full potential. We simply do not know what kind of data exists out there, what it contains, or how to use it without a huge investment with respect to time and manpower. 
```

#Aim of this project
The overarching aim of the present work is to make optimal use of the substantial potential of open data in the field. To achieve this aim, we first provide detailed insights into data sharing practices in the field of fear conditioning research in humans. We evaluate to what extend publicly shared data-sets are functional and work out clear field specific guidelines which can be expected to not only be broadly applicable also to other fields of psychology. 
Second, based on this evaluation and inventory, we work out clear and easy-to-implement suggestions with respect to data structure and onotonogy for variable names as well as meta-data. Such as homogenization is a necessary pre-requirement for future re-use and merging of datasets and the worked out recomendations are based  community-driven discussions [ERSTMAL VERMUTUNG] as well as published recommendations for data sharing and data management (e.g., @Gollwitzer (2020). 

> Finally, we aim to  generate a large-scale database of open data in fear conditioning research (FEAR-BASE) to facilitate meta and mega analyses. This aim rests fundamentally on a cooperative spirit of data sharing to maximizes what the scientific community can learn from the data acquired by an individual researcher (@Gollwitzer(2020)).

Reviewer: ```suggestion
Finally, we aim to generate a large-scale database of open data in fear conditioning research (FEAR-BASE) to facilitate meta and mega analyses. This aim rests fundamentally on a cooperative spirit of data sharing to maximize what the scientific community can learn from the data acquired by an individual researcher (@Gollwitzer(2020)).
```
